### PR TITLE
Update wf_crm_admin_form.inc to fix a wrong link generation in a help message

### PR DESCRIPTION
**Pull request by uniquename**
Overview
----------------------------------------
Fixed a wrong link generation in a help message

Before
----------------------------------------
Link was not created properly leading to a non working a tag.

After
----------------------------------------
Link is create in a corrext way.

### CODE REVIEW

File: includes/wf_crm_admin_form.inc [hunk 1]
```diff
@@ -1525,8 +1526,10 @@ class wf_crm_admin_form {
           'civicrm_' . $c . '_contact_1_contact_first_name' => 'create_civicrm_webform_element',
           'civicrm_' . $c . '_contact_1_contact_last_name' => 'create_civicrm_webform_element',
         ];
-        $link = [':link' => 'href="https://docs.civicrm.org/sysadmin/en/latest/integration/drupal/webform/#cloning-a-contact" target="_blank"'];
-        \Drupal::messenger()->addStatus(t('Tip: Consider using the clone feature to add multiple similar contacts. (<a :link>more info</a>)', $link));
+        $options = array('attributes' => array('target' => '_blank'));
+        $url = Url::fromUri('https://docs.civicrm.org/sysadmin/en/latest/integration/drupal/webform/#cloning-a-contact', $options);
+        $link = Link::fromTextAndUrl('more info', $url)->toString();
```
Comment (colemanw) [inline includes/wf_crm_admin_form.inc]: Should 'more info' be wrapped in `t()` here?

Comment (uniquename) [reply includes/wf_crm_admin_form.inc]: Yes, it should.

File: includes/wf_crm_admin_form.inc [hunk 2]
```diff
@@ -1525,8 +1526,10 @@ class wf_crm_admin_form {
           'civicrm_' . $c . '_contact_1_contact_first_name' => 'create_civicrm_webform_element',
           'civicrm_' . $c . '_contact_1_contact_last_name' => 'create_civicrm_webform_element',
         ];
-        $link = [':link' => 'href="https://docs.civicrm.org/sysadmin/en/latest/integration/drupal/webform/#cloning-a-contact" target="_blank"'];
-        \Drupal::messenger()->addStatus(t('Tip: Consider using the clone feature to add multiple similar contacts. (<a :link>more info</a>)', $link));
+        $options = array('attributes' => array('target' => '_blank'));
+        $url = Url::fromUri('https://docs.civicrm.org/sysadmin/en/latest/integration/drupal/webform/#cloning-a-contact', $options);
```
Comment (mglaman) [inline includes/wf_crm_admin_form.inc:1530]: URL generation is not required, this HTML could go into a FormattableMarkup string (like sprintf style)

**Review by mglaman — COMMENTED** [review]
This could be simplified to just the HTML string in the formattable markup, there is no need to use URL generation since it is a remote URL.

File: includes/wf_crm_admin_form.inc (hunk 2 above)
Comment (uniquename) [reply includes/wf_crm_admin_form.inc:1530]: well, just found it not working at all, so i wanted to give a quick fix. don't have more time...

**Comment by mglaman** [comment]
:( I didn't mean to discourage, @uniquename I just was providing some feedback so we could provide a fix and clean up the existing code.

**Pull request closed without merging**